src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -236,15 +249,15 @@ private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activ
             {
                 currentCancellationToken.ThrowIfCancellationRequested();
 
-                // if we were in an error state retain that,
+                // if we were in an error State retain that,
                 // otherwise move to default
-                if (State != ScriptHostState.Error)
+                if (_state != ScriptHostState.Error)
                 {
-                    State = ScriptHostState.Default;
+                    _state = ScriptHostState.Default;
```

Comment:
Why are you bypassing the property setter with these changes - you'll be bypassing the whole point of your new log statement? You should be using the property throughout.

Reply:
Woops! Accident, will fix!!